classes/Tools.php

```diff
@@ -1428,13 +1430,100 @@ public static function str2url($str)
      */
     public static function replaceAccentedChars($str)
     {
-        static $transliterator;
+        /* One source among others:
```

Comment:
I think there was a glitch during your rebase, you seem to be reintroducing some methods which have been removed or change on develop

Reply:
`replaceAccentedChars` has been modified in this PR https://github.com/PrestaShop/PrestaShop/pull/27176

Reply:
yes I revert this ! Thx